docs/building-a-form/common-definitions.md

```diff
@@ -61,83 +66,119 @@ schema: {
         'New York',
         'Chicago'
       ]
+      }
     }
   }
 },
 uiSchema: {
-  officeLocation: autosuggestUI(
-    'Preferred Office Location',  // field title
-    null,         // Promise to get options (optional)
-    {             // Additional uiSchema options
-      'ui:options': {
-        // When labels are not provided, it uses enumNames
-        labels: { }
-      }
-    }
+  officeLocation: autosuggestConfig.uiSchema(
+    ...
```

Comment:
Oops, sorry! I think my previous feedback was confusing. This is what the code should look like here:

```
uiSchema: {
  officeLocation: autosuggestConfig.uiSchema(
    'Preferred Office Location',  // field title
    null,         // Promise to get options (optional)
    {             // Additional uiSchema options
      'ui:options': {
        // When labels are not provided, it uses enumNames
        labels: { }
      }
    }
  )
}
```

Reply:
Many thanks!